### ISSUE TITLE
RHEL-9.0: Drop `grub2-efi-ia32` sub-packages from image package sets

### ISSUE THREAD

**Issue by thozza** [osbuild/osbuild-composer#2181]
**Describe the bug**
32bit UEFI packages are being dropped from RHEL-9.0 repos, therefore we should not list them in any default image type package sets.

This affected only the `image-installer` image type, last time I checked.

```
$ grep -r grub2-efi-ia32
rhel84/distro.go:               "grub2-efi-ia32-cdboot",
rhel84/distro.go:               "grub2-efi-ia32-cdboot",
rhel90/package_sets.go:                         "grub2-efi-ia32-cdboot",
rhel90/package_sets.go:                 "grub2-efi-ia32-cdboot",
rhel85/package_sets.go:                         "grub2-efi-ia32-cdboot",
rhel86/package_sets.go:                         "grub2-efi-ia32-cdboot",
rhel90beta/package_sets.go:                     "efibootmgr", "grub2-efi-ia32-cdboot",
rhel90beta/package_sets.go:                     "google-noto-sans-cjk-ttc-fonts", "grub2-efi-ia32-cdboot",
rhel90beta/package_sets.go:                     "google-noto-sans-cjk-ttc-fonts", "grub2-efi-ia32-cdboot",
```

**Comment by runcom** [comment]
I've done it for a couple of them in a commit here https://github.com/osbuild/osbuild-composer/pull/1884

**Comment by thozza** [comment]
Resolved by #2219
